icd/api/include/compiler_solution.h

```diff
@@ -111,6 +112,7 @@ struct ComputePipelineCreateInfo
     bool                                   freeWithCompiler;
     Util::MetroHash::Hash                  basePipelineHash;
     PipelineCreationFeedback               pipelineFeedback;
+    PipelineCreationFeedback stageFeedback;
```

Comment:
the field name should be aligned to the right

Reply:
done